src/main.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+main() {
+    args=( $(find ./src/programs -type f | sed 's!.*/!!' | tr '\n' ' ') "${@}" )
+    sudo xbps-install -y "${args[@]}"
+}
+
+echo "Install programs.."
+main
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Use main "$@" if function's $1 should mean script's $1. [SC2119](https://github.com/koalaman/shellcheck/wiki/SC2119)